dockstore-webservice/pom.xml

```diff
@@ -1197,7 +1197,7 @@
                         <package>io.dockstore.webservice.resources.proposedGA4GH</package>
                     </resourcePackages>
                     <objectMapperProcessorClass>io.swagger.OrderedObjectMapperProcessor</objectMapperProcessorClass>
-                    <readAllResources>false</readAllResources>
```

Comment:
The first significant change. This is why the v1 and v2 beta endpoints were not in the openapi.yaml, they were never annotated with openapi 3 annotations